cmd/generic-handlers.go

```diff
@@ -550,6 +550,9 @@ func addCustomHeaders(h http.Handler) http.Handler {
 		// part of the log entry, Error response XML and auditing.
 		// Set custom headers such as x-amz-request-id for each request.
 		w.Header().Set(xhttp.AmzRequestID, mustGetRequestID(UTCNow()))
+		if globalLocalNodeName != "" {
+			w.Header().Set(xhttp.AmzRequestNodeID, globalLocalNodeName)
```

Comment:
Isn't this considered a little bit of sensitive information, we can calculate its hash once when the node is started and use it instead.

Reply:
why is it sensitive?

Reply:
We are sending CPU information in update check, this is less sensitive than that :-) @vadmeste